### DISCUSSION TITLE
Get configAudit ID in Prometheus

### DISCUSSION THREAD

**Discussion by JKnino** [aquasecurity/trivy-operator#1146] [Help & Support] [answered]
would like to get also KSV IDs in Prometheus

**Comment by chen-keinan** [comment, marked as answer]
@JKnino the way to achieve it is to add a similar functionality same we have [vulnerability CVE](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/values.yaml#L90)  for configAudit report.

could you please open issue for it

**Reply by JKnino** [reply]
will do, thank you
